daemon/api.go

```diff
@@ -661,6 +661,7 @@ func postSnap(c *Command, r *http.Request, user *auth.UserState) Response {
 
 	chg := newChange(state, inst.Action+"-snap", msg, tsets)
 	chg.Set("snap-names", []string{inst.snap})
+	chg.Set("api-data", map[string]string{"snap-name": inst.snap})
```

Comment:
This looks strange. I think we should either have one or the other, based on whether  the operation operates on multiple snaps or a single one. Having both most likely means one of them is wrong.